gui/wxpython/vdigit/mapwindow.py

```diff
@@ -689,13 +689,6 @@ def OnLeftDownUndo(self, event):
 
     def _onLeftDown(self, event):
         """Left mouse button donw - vector digitizer various actions"""
-        try:
-            mapLayer = self.toolbar.GetLayer().GetName()
-        except:
-            GMessage(parent=self, message=_("No vector map selected for editing."))
-            event.Skip()
-            return
-
```

Comment:
I'm not sure about this one. This looks like a pattern to cancel an event, by testing a condition that would allow to continue the function if doesn't fail

Reply:
I agree.